src/main/java/net/snowflake/ingest/SimpleIngestManager.java

```diff
@@ -246,38 +302,28 @@ public SimpleIngestManager(String account, String user, String pipe, PrivateKey
   }
 
   /**
-   * Constructs a SimpleIngestManager for a given user in a specific account In addition, this also
-   * takes takes the target table and source stage Finally, it also requires a valid KeyPair object
-   * registered with Snowflake DB
+   * Using this constructor for Builder pattern. KeyPair can be passed in now since we have
+   * made @see {@link SimpleIngestManager#createKeyPairFromPrivateKey(PrivateKey)} public
    *
-   * <p>This method is deprecated, please use the constructor that only requires PrivateKey instead
-   * of KeyPair.
-   *
-   * @param account the account into which we're loading Note: account should not include region or
-   *     cloud provider info. e.g. if host is testaccount.us-east-1.azure .snowflakecomputing.com
-   *     account should be testaccount If this is the case, you should use the constructor that
-   *     accepts hostname as argument
-   * @param user the user performing this load
-   * @param pipe the fully qualified name of the pipe
-   * @param keyPair the KeyPair we'll use to sign JWT tokens
-   * @param schemeName http or https
-   * @param hostName the hostname
-   * @param port the port number
+   * @param account
+   * @param user
+   * @param pipe
+   * @param hostName
+   * @param keyPair
+   * @param userAgentSuffix
```

Comment:
Would be good if add some comments for these fields in each constructor? Might be useful when users want to show method signatures in IDE.

Reply:
Will do..